spec/player_turn_spec.rb

```diff
@@ -1,33 +1,20 @@
 require_relative "../lib/player_turn"
 
 RSpec.describe PlayerTurn do
-  describe "#get_coordinates" do
-    it "returns the user's input" do
-      input = double
-      mock_view = double
-      player_turn = PlayerTurn.new(input: input, view: mock_view)
-      allow(mock_view).to receive(:render_coordinates_prompt)
-      allow(mock_view).to receive(:render_coordinates_error)
-      allow(input).to receive(:gets).and_return("A1")
+  describe "#mark_grid" do
+    it "gets payer input and updates the grid" do
```

Comment:
This spec stubs out a bit more than I am comfortable with. Since the test seems to be about the grid, I would leave that object as a real thing and just stub out the player input. That would put this spec into the middle layer of the pyramid, whereas it is in the bottom right now. As an object that does a massive amount of coordinating between collaborators, this is a good place to exercise some of the collaborators to ensure it really works.